### PR TITLE
Update data source docs

### PR DESCRIPTION
**Pull request by warittornc**

### CODE REVIEW

File: docs/custom-script/data-source/deployment.md [hunk 1]
```diff
@@ -2,65 +2,193 @@
...
-    --fee 250000uband # fee charged when using this data source
-    --description "CoinGecko with symbols mapping" # description of the data source
-    --script tutorial_data_source.py # path to the data source script
-    --from account01 # sender account
-    --owner band1v44t7mrda70yyv0sxl7kpnmtmh07wn9x87rft3 # data source owner address
-    --treasury band1v44t7mrda70yyv0sxl7kpnmtmh07wn9x87rft3 # address to collect the fee
-    --node http://rpc-laozi-testnet4.bandchain.org:26657 # rpc node
-    --chain-id band-laozi-testnet4 # chain id
-    --gas auto # transaction gas
+A MsgCreateDataSource message contains various parameters of the data source they wish to register. These parameters
```
Comment (babybunn) [inline docs/custom-script/data-source/deployment.md]: MsgCreateDataSource change to `MsgCreateDataSource`

File: docs/custom-script/data-source/deployment.md [hunk 2]
```diff
@@ -2,65 +2,193 @@
...
+
+[//]: # (- Per-query fee that someone looking to use that data source needs to pay)
+
+[//]: # (- Name of the data source)
+
+[//]: # (- Content of the executable to be run by block validators upon receiving a data request for this data source)
+
+[//]: # (- Sender who wishes to create the data source)
+
+[//]: # (- Owner of the data source, if specified)
```
Comment (pzshine) [inline docs/custom-script/data-source/deployment.md]: Please fix this

Comment (warittornc) [reply docs/custom-script/data-source/deployment.md]: old comment, removed now

File: docs/custom-script/data-source/deployment.md (hunk 1 above)
Comment (warittornc) [reply docs/custom-script/data-source/deployment.md]: fixed

**Review by pzshine — APPROVED** [review]

**Pull request merged**